lib/src/parse/stylesheet.dart

```diff
@@ -1403,6 +1403,12 @@ abstract class StylesheetParser extends Parser {
         break;
 
       case $0:
+        if (scanner.peekChar(1) == $backslash && scanner.peekChar(2) == $0) {
```

Comment:
these lines are for parsing code like next:

```scss
@media screen and (min-width:0\0) {
  ...
}
```

specifically are for parsing `0\0` inside a media query.